ChaosMod/Memory/Drawing.h

```diff
@@ -0,0 +1,83 @@
+#pragma once
+
+#include "Util/Color.h"
+
+// Thanks CitizenFX!
+// Adapted by MoneyWasted for ChaosModV
+// https://github.com/citizenfx/fivem/blob/master/code/components/extra-natives-five/src/Draw2dNatives.cpp
+
+// This file is manually formatted.
+// clang-format off
+
+constexpr int MAX_LINES = 5000;
+
+char *(*OG_AllocateDrawRect)(void *);
+void (*OG_SetDrawRectCoords)(void *, float, float, float, float);
+
+namespace Hooks
+{
+	inline void DrawLine(float x1, float y1, float x2, float y2, float width, int r, int g, int b, int a)
+	{
+		static char *ms_DrawRects             = 0;
+		static int ms_DrawRectsSize           = 0;
+		static int *ms_MainThreadFrameIndex   = 0;
+
+		static void *ms_AllocateDrawRectAddr  = 0;
+		static void *ms_SetDrawRectCoordsAddr = 0;
```

Comment:
```suggestion
		static char *drawRects             = 0;
		static int drawRectsSize           = 0;
		static int *mainThreadFrameIndex   = 0;

		static void *allocateDrawRectAddr  = 0;
		static void *setDrawRectCoordsAddr = 0;
```
We don't do prefixes in this case. Again a nitpick but outside of effect files I have to be a bit more stingy for consistency purposes